src/i18n/rpi-imager_fr.ts

```diff
@@ -386,7 +386,7 @@
     <message>
         <location filename="../OptionsPopup.qml" line="222"/>
         <source>Username:</source>
-        <translation>Nom d&apos;utilisateur :</translation>
+        <translation>Nom d&apos;utilisateur&nbsp;:</translation>
```

Comment:
I'm not a French-speaker, but just out of curiosity is there a reason why the French translations all have a space before the colon, when this isn't present in the original English labels?

Reply:
There is indeed. In French typography, double punctuation marks (";", ":", "?" and "!") must be preceded by a non-breaking space and followed by another space. See eg. [Wikipedia rules](https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Conventions_typographiques#Signes_de_ponctuation):

> les signes de ponctuation doubles (« ; », « : », « ? » et « ! ») doivent être précédés d’une espace insécable et suivis d'une autre espace (...)

And it is frequently a challenge for localization of software.